datalad/support/annexrepo.py

```diff
@@ -3018,8 +2901,8 @@ def _format_response(res):
                 files = [files]
             # anything else is assumed to be an iterable (e.g. a generator)
         if batch is False:
-            for res in self._run_annex_command_json(
-                    'metadata', opts=['--json'], files=files):
+            for res in self._call_annex_records(
+                    ['metadata', '--json'], files=files):
```

Comment:
Doesn't functionally matter and is true for the old code too, but since it's being converted: this leads to git-annex being called with two `--json` flags.

Reply:
Done